relay/polkadot/src/lib.rs

```diff
@@ -3404,7 +3378,9 @@ mod multiplier_tests {
 		let mut fees_paid = 0;
 
 		frame_system::Pallet::<Runtime>::set_block_consumed_resources(Weight::MAX, 0);
-		let info = DispatchInfo { weight: Weight::MAX, ..Default::default() };
+		// TODO: Find out if this test is correct, since we're not yet considering
+		// `extension_weight`
+		let info = DispatchInfo { call_weight: Weight::MAX, ..Default::default() };
```

Comment:
```suggestion
		let info = DispatchInfo { call_weight: Weight::MAX, ..Default::default() };
```

Yes we are good, the info is used to track the fees and put in the log, the `extension_weight` is not important here.

Reply:
See https://github.com/polkadot-fellows/runtimes/pull/606#discussion_r2029078959